Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -707,7 +707,7 @@ private final class TokenStreamCreator: SyntaxVisitor {
 
   func visit(_ node: ArrayElementListSyntax) -> SyntaxVisitorContinueKind {
     insertTokens(.break(.same), betweenElementsOf: node)
-    if let firstElement = node.first, let lastElement = node.last {
+    if let firstElement = node.first, let lastElement = node.last, firstElement != lastElement {
```

Comment:
Can you comment before this line (and in the dictionary function) why the `first != last` check is important so that future contributors (and future _us_) have that information? It's a little less discoverable below the fold of a commit message in the git history.

Reply:
Good idea. This isn't the most obvious problem and solution.